proposed/container-meta.md

```diff
@@ -239,7 +239,16 @@ While `ContainerInterface` only defines one mandatory parameter in `get()`, it i
 existing containers that have additional optional parameters. PHP allows an implementation to offer more parameters
 as long as they are optional, because the implementation *does* satisfy the interface.
 
-This issue has been discussed in [container-interop's issue #6](https://github.com/container-interop/container-interop/issues/6).
+Difference with container-interop: [The container-interop spec](https://github.com/container-interop/container-interop/blob/master/docs/ContainerInterface.md) stated that:
+
+> While `ContainerInterface` only defines one mandatory parameter in `get()`, implementations MAY accept additional optional parameters.
+
+This sentence was removed from PSR-11 because:
+
+- It is something that stems from OO principles in PHP, so this is not directly related to PSR-11
+- We do not want to encourage implementors to add additional parameters as we recommend coding against the interface and not the implementation
+
+Yet, some implementations have extra optional parameters; that's technically legal. Such implementations are compatible with PSR-11. [[6]](#link_get_optional_parameters)
```

Comment:
That sounds good to me, with the caveat of only including links if the metadoc itself cannot reasonably summarize the points in them.  (That's a very subjective guideline, I know.)  *If* offsite links are needed, put them at the end.  If we can avoid them and summarize effectively, that's even better.